package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "mjml-theme-converter",
+  "version": "1.0.0",
+  "description": "mjml-theme-converter ====================",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lmeyer1/mjml-theme-converter.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Must be OSL https://github.com/PrestaShop/mjml-theme-converter/blob/master/LICENSE.md

Reply:
Sorry, this was just the output of `npm init`, without any adaptation.